### ISSUE TITLE
dom-event@0.0.5 no longer exists

### ISSUE THREAD

**Issue by gsound** [royriojas/dom-event-special#1]
`dom-event` package was unpublished and then republished as 2.0.0. Update its dependencies please.

```
npm ERR! No compatible version found: dom-event@0.0.5
npm ERR! Valid install targets:
npm ERR! ["2.0.0","0.0.1-security"]
```

**Comment by royriojas** [comment]
@gsound the module added is just a placeholder, I replaced it locally. it should be fine now.

**Comment by royriojas** [comment]
published as `v0.1.7`

**Comment by gsound** [comment]
Don't forget to publish it.
